crates/fluvio/Cargo.toml

```diff
@@ -40,7 +40,11 @@ siphasher = "0.3.5"
 cfg-if = "1.0.0"
 derive_builder = "0.10"
 async-trait = "0.1.51"
-
+chrono = "0.4"
```

Comment:
These dependencies should be under separate feature like `stat`?

Reply:
I ended up getting rid of chrono. I was only using it in one place for timestamps, so I replaced it